packages/orchestrator-ui-components/src/components/WfoTextAnchor/styles.ts

```diff
@@ -0,0 +1,17 @@
+import { EuiThemeComputed } from '@elastic/eui/src/services/theme/types';
+import { css } from '@emotion/react';
+
+export const getStyles = (theme: EuiThemeComputed) => {
+    const textAnchorStyle = css({
+        marginTop: 2,
```

Comment:
The theme has a value for this:
size.xxs: '2px',

Reply:
Fixed